HealthSDK/GiniHealthSDK/Package-release.swift

```diff
@@ -0,0 +1,32 @@
+// swift-tools-version:5.5
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "GiniHealthSDK",
+    defaultLocalization: "en",
+    platforms: [.iOS(.v11)],
+    products: [
+        // Products define the executables and libraries a package produces, and make them visible to other packages.
+        .library(
+            name: "GiniHealthSDK",
+            targets: ["GiniHealthSDK"]),
+    ],
+    dependencies: [
+        // Dependencies declare other packages that this package depends on.
+        // .package(url: /* package url */, from: "1.0.0"),
+        .package(name: "GiniHealthAPILibrary", url: "https://github.com/gini/health-api-library-ios.git", from: "0.0.1"),
```

Comment:
Here we should also use an exact version to have deterministic version resolution for our packages.